syntax = "proto3";

message AddressNode{
  string code = 1;
  string name = 2;
  map<string, string> childs = 3;
}

message AddressConf {
  map<string, AddressNode> code2addr = 1;
}
